src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1336,24 +1335,24 @@ private void startContainer() {
                 runCmd("mkdir -p " + buildDirectory + "/" + DEVC_HIDDEN_FOLDER + "/dropins");
             }
 
-            info("Starting Docker container...");
+            info("Starting container...");
             String startContainerCommand = getContainerCommand();
             info(startContainerCommand);
-            dockerRunProcess = getRunProcess(startContainerCommand);
-            execDockerCmdAndLog(dockerRunProcess, 0);
+            containerRunProcess = getRunProcess(startContainerCommand);
+            execContainerCmdAndLog(containerRunProcess, 0);
         } catch (IOException e) {
             error("Error starting container: " + e.getMessage());
             throw new RuntimeException(e);
         } catch (InterruptedException e) {
             error("Thread was interrupted while starting the container: " + e.getMessage());
         } catch (RuntimeException r) {
             try {
-                // remove container in case of an error trying to run the container because the docker run --rm will not rm the container
-                String dockerRmCmd = "docker container rm " + containerName;
-                execDockerCmd(dockerRmCmd, DOCKER_TIMEOUT);
+                // remove container in case of an error trying to run the container because the container run --rm will not rm the container
+                String containerRmCmd = "container rm " + containerName;
+                execContainerCmdWithPrefix(containerRmCmd, DOCKER_TIMEOUT);
```

Comment:
I should have caught this before. Make `CONTAINER_TIMEOUT` public in `AbstractContainerSupportUtil` and use it throughout this class. Get rid of `DOCKER_TIMEOUT`.